pennylane/operation.py

```diff
@@ -804,7 +813,9 @@ def label(self, decimals=None, base_label=None, cache=None):
 
         if len(qml.math.shape(params[0])) != 0:
             # assume that if the first parameter is matrix-valued, there is only a single parameter
-            # this holds true for all current operations and templates
+            # this holds true for all current operations and templates unless tensor-batching
+            # is used
+            # TODO[dwierichs]: Implement a proper label for tensor-batched operators
```

Comment:
What does this todo mean?

Reply:
> Overall, I am not sure what the best approach of drawing broadcasted tapes

Oh good question; what happens right now when you draw a broadcasted tape with `decimals=...`?

Reply:
Sorry, my last comment above with the example was pending because I didn't hit send... :grimacing:

Reply:
ah I see! makes sense, since the drawing logic is assuming anything with dimensions is a large matrix.

This is tricky; we probably want to keep this for _large_ batches of parameters, but it would be nice to print small batches of parameters.

Reply:
Yes, indeed :thinking: I'll open an issue for this as well :)

Reply:
thanks!